### PR TITLE
[Security] Missing Bulgarian translations

### LINKED ISSUE #38730: Missing translations for Bulgarian (bg)
Hello,

There are some missing translation for Bulgarian (bg). These should be added in branch 3.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.bg.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Form/Resources/translations/validators.bg.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.bg.xlf](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Validator/Resources/translations/validators.bg.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by kbosilkov**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38730
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf [hunk 1]
```diff
@@ -66,6 +66,14 @@
                 <source>Account is locked.</source>
                 <target>Акаунта е заключен.</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>Твърде много грешни опити за вход, моля опитайте по-късно.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf:71]: Does it say "Too many **failed** login attempts"?

**Review by Nyholm — COMMENTED** [review]
Thank you. 
I just have one small question

File: src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf (hunk 1 above)
Comment (kbosilkov) [reply src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf:71]: Yes, "Твърде много **грешни** опити за вход"

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf:71]: Okey. Awesome. 

(Google translate cannot be trusted) =)

**Review by Nyholm — APPROVED** [review]
Excellent job with this 3 PRs. Thank you!

File: src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf (hunk 1 above)
Comment (kbosilkov) [reply src/Symfony/Component/Security/Core/Resources/translations/security.bg.xlf:71]: For sure :)

**Comment by derrabus** [comment]
Thank you @kbosilkov.

**Pull request merged**